guides/coco-dev.md

```diff
@@ -0,0 +1,248 @@
+
+In this example, we will be using the bitnami/nginx image as described in the following yaml:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: nginx
+  name: nginx
+  annotations:
+    io.containerd.cri.runtime-handler: kata-qemu-coco-dev
+spec:
+  containers:
+  - image: bitnami/nginx:1.22.0
+    name: nginx
+  dnsPolicy: ClusterFirst
+  runtimeClassName: kata-qemu-coco-dev
+```
+
+Setting the `runtimeClassName` is usually the only change needed to the pod yaml, but some platforms
+support additional annotations for configuring the enclave. See the [guides](../guides) for
+more details.
+
+With Confidential Containers, the workload container images are never downloaded on the host.
+For verifying that the container image doesn’t exist on the host, you should log into the k8s node and ensure the following command returns an empty result:
+```
+root@cluster01-master-0:/home/ubuntu# crictl -r unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
+```
+You will run this command again after the container has started.
+
+Create a pod YAML file as previously described (we named it `nginx.yaml`) .
+
+Create the workload:
+```
+kubectl apply -f nginx.yaml
+```
+Output:
+```
+pod/nginx created
+```
+
+Ensure the pod was created successfully (in running state):
+```
+kubectl get pods
+```
+Output:
+```
+NAME    READY   STATUS    RESTARTS   AGE
+nginx   1/1     Running   0          3m50s
+```
+
+Now go back to the k8s node and ensure that you don’t have any bitnami/nginx images on it:
+```
+root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
+```
+
+## Encrypted and/or signed images with attestation
+
+The previous example does not involve any attestation because the workload container isn't signed or encrypted,
+and the workload itself does not require any secrets.
+
+This is not the case for most real workloads. It is recommended to use CoCo with signed and/or encrypted images.
+The workload itself can also request secrets from the attestation agent in the guest.
+
+Secrets are provisioned to the guest in conjunction with an attestation, which is based on hardware evidence.
+The rest of this guide focuses on setting up more substantial encrypted/signed workloads using attestation
+and confidential hardware.
+
+CoCo has a modular attestation interface and there are a few options for attestation.
+CoCo provides a generic Key Broker Service (KBS) that the rest of this guide will be focused on.
+The SEV runtime class uses `simple-kbs`, which is described in the [SEV guide](../guides/sev.md).
+
+### Select Runtime Class
+
+To use CoCo with confidential hardware, first switch to the appropriate runtime class.
+TDX has one runtime class, `kata-qemu-tdx`. 
+
+For SEV(-ES) use the `kata-qemu-sev` runtime class and follow the [SEV guide](../guides/sev.md). 
+
+For SNP, use the `kata-qemu-snp` runtime class and follow the SNP guide.
```

Comment:
```suggestion
For SNP, use the `kata-qemu-snp` runtime class.
```

or do we have it?

Reply:
We will have it in pr 237 that will be worked on and merged after this PR is submitted.